changelog.json

```diff
@@ -1,5 +1,13 @@
 {
   "releases": {
+    "1.2.3": [
+      "[Fixed] No autocomplete when searching for co-authors - #4847",
+      "[Fixed] Error when checking out a PR from a fork - #4842"
+    ],
+    "1.2.3-beta1": [
```

Comment:
Maybe I am wrong but isn't this technically beta0 as it is the same as what is `1.2.3`?

Reply:
@say25 yeah, it's in the [release process](https://github.com/desktop/desktop/blob/master/docs/process/releasing-updates.md#3-releasing) but oh well - version numbers are cheap.